src/k8s/cmd/k8s/k8s_bootstrap.go

```diff
@@ -42,6 +42,10 @@ var (
 				config.SetDefaults()
 			}
 
+			if !config.IsValidCIDR() {
```

Comment:
Should we just have server-side validation? 
Seems cumbersome to implement those validation steps on all clients (e.g. the charm would also need to do this).

Just check on the server and send back a validation error.
(We can also do this as a follow-up step.)